search-app/src/cmr/search/services/query_execution/multi_part_query_feature_common.clj

```diff
@@ -42,6 +42,7 @@
   [query granule-field]
   (-> query
       (dissoc :result-features)
+      (dissoc :result-format)
```

Comment:
I see the following in `cmr.common-app.services.search.params/default-parse-query-level-params`:
`(dissoc params :offset :page-size :page-num :result-format :scroll :sort-key)`

I'm having trouble following out how this all works and of course that function doesn't have a docstring, but should we be removing all of these parameters for this query as well?